Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb

```diff
@@ -48,11 +48,20 @@
    "outputs": [],
```

Comment:
<p>add a comment about the black line (deseasonalised monthly mean) vs the grey lines (seasonal monthly means)? potentially consider a legend on the RH plot, though dunno where it'd fit...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/724/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>